.github/workflows/issue-triage.md

```
@@ -51,35 +78,38 @@ You're a triage assistant for GitHub issues. Your task is to analyze issues crea
-   - Search for similar issues, and if you find similar issues consider using a "duplicate" label if appropriate. Only do so if the issue is a duplicate of another OPEN issue.
-   - Only select labels from the provided list above
+   - Search for similar issues. If you find a duplicate of another OPEN issue in THIS repository, you may use a "duplicate" label (if available) and reference the canonical issue.
+   - If the closest match is in another repository within the Appwrite org, do NOT mark as duplicate here; instead, link it in your comment under a "Cross‑repo related issues" section.
+   - Only select labels from the provided list
+   - Don't apply the `good first issue` or `help wanted` labels
    - It's okay to not add any labels if none are clearly applicable
 
-9. Apply the selected labels:
+10. Apply the selected labels:
 
    - Use the `update_issue` tool to apply the labels to the issue
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)